### DISCUSSION TITLE
Where to define type of ESP32-Cam

### DISCUSSION THREAD

**Discussion by bernardtrigaux** [jomjol/AI-on-the-edge-device#3156] [General]
Hello,

I have a ESP32 WROVER from Freenove. 
It doesn't succeed to initiate the camera.
As far as I understand, I think the build available in Release is for AI-THINKER.
Does it mean that I should rebuild for CAMERA_MODEL_WROVER_KIT ?
If so where should this CAMERA_MODEL_WROVER_KIT be best declared ?
Question from a newbie on building ESP32 applications...

Thanks !

**Comment by caco3** [comment]
What is written in the log?

**Reply by SybexX** [reply]
https://github.com/jomjol/AI-on-the-edge-device/discussions/3007#discussioncomment-10242490
